src/datePicker/DatePicker.tsx

```diff
@@ -39,6 +39,7 @@ export interface DatePickerProps {
   readonly toMonth?: Date
   readonly firstDayOfWeek?: number
   readonly stickyPositionTop?: number
+  readonly focus?: boolean
```

Comment:
Is this prop still used? I didn't find it's usages 🤔.

Reply:
Yes passing it from the DatePickerOverlay.

Reply:
Just added in the condition to focus the datepicker